dojo/tools/netsparker/parser.py

```diff
@@ -20,26 +19,27 @@ def get_description_for_scan_types(self, scan_type):
     def get_findings(self, filename, test):
         tree = filename.read()
         try:
-            data = json.loads(str(tree, 'utf-8-sig'))
-        except:
+            data = json.loads(str(tree, "utf-8-sig"))
+        except BaseException:
```

Comment:
Change to `Exception`

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8305/commits/5cc690eda741b1f58c6e5d39075007108035d7a7